pages/linux/goldeneye.md

```diff
@@ -0,0 +1,24 @@
+# goldeneye
+
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
+
+`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
+
+- Test a specific website without verifying ssl certficate:
+
+`./goldeneye.py {{url}} -n"`
+
+- Test a specific website in debug mode:
+
+`./goldeneye.py {url}} -d"`
```

Comment:
The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --debug`
```

- Unbalanced closing `}` and trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).